test_runner/src/main/kotlin/ftl/args/ArgsHelper.kt

```diff
@@ -45,6 +48,24 @@ object ArgsHelper {
         }
     }
 
+    fun evaluateFilePath(filePath: String): String {
+        var file = filePath.trim().replaceFirst("~", System.getProperty("user.home"))
+        file = substituteEnvVars(file)
+        // avoid File(..).canonicalPath since that will resolve symlinks
+        file = Paths.get(file).toAbsolutePath().normalize().toString()
+
+        // Avoid walking the folder's parent dir if we know it exists already.
+        if (File(file).exists()) return file
```

Comment:
I would prefer if we also printed this value to console. Something like - `Resolved app file path - "$file"`.
Under a conditional. `if (filePath != file)`.

Reply:
I think it should be printed already when the args are printed to stdout at the start of a test run.